Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -1197,6 +1197,10 @@ def main():  # pragma: no cover
     demisto.debug(f'Command being called is {command}')
 
     try:
+        if params.get("isFetchEvents", False) and params.get("longRunning", False):
```

Comment:
lets only put this check under the commands - it shouldnt fail the other commands